docs/BigQueryTable-batchsink.md

```diff
@@ -23,6 +23,12 @@ Properties
 ----------
 **Reference Name:** Name used to uniquely identify this sink for lineage, annotating metadata, etc.
 
+**Use Connection:** Whether to use a connection, if a connection is used,
+the credentials does not need to be provided.
+
+**Connection:** Name of the connection to use, should use the macro function ${conn:(connection-name)} to provide.
```

Comment:
same here , no colon in the connection macro

Reply:
done